content/data/digilocker/error-codes.mdx

```diff
@@ -89,14 +92,50 @@ Below is a table which contains the error codes and corresponding error messages
             <td>
                 <code>upstream_server_error</code>
             </td>
-            <td> Document Repository Service Error <br/> Partner Service Error <br/> Aadhaar Information not available <br/> Upstream service failure </td>
+            <td> Upstream service failure <br/> Document Repository Service Error <br/> Partner Service Error <br/> Invalid response from issuer [#ISR400] </td>
         </tr>
         <tr>
             <td>
                 <code>internal_server_error</code>
             </td>
             <td> Internal server error </td>
         </tr>
+        <tr>
+            <td>
+                <code>customer_detail_mismatch</code>
+            </td>
+            <td> Mismatch in customer data (e.g. dob/name) across documents (e.g. aadhaar, pan) </td>
+        </tr>
+        <tr>
+            <td>
+                <code>invalid_uri</code>
+            </td>
+            <td> No file found for given URI </td>
+        </tr>
+        <tr>
+            <td>
+                <code>aadhaar_not_available</code>
+            </td>
+            <td> Aadhaar data is not available for this user. Please perform Aadhaar eKYC again. </td>
```

Comment:
Should this be eKYC or KYC?

Reply:
It was like this on the digilocker doc. I will make it to KYC.